CMSgov/design-system
# [WNMGDS-2864] Add utility actions pattern guidance page

## Summary

[Ticket](https://jira.cms.gov/browse/WNMGDS-2864) 

Added new guidance page for the utility actions pattern.

**Note**
The design will differ on this branch because the EmailIcon is being imported from the local Doc Site component rather than the Design System. [A ticket is up to fix this issue](https://jira.cms.gov/browse/WNMGDS-2915?atlLinkOrigin=c2xhY2staW50ZWdyYXRpb258aXNzdWU%3D). 

## How to test

1. Run the doc site locally, or check the [deployed demo url](https://cmsgov.github.io/design-system/branch/utility-action-demo/patterns/utility-actions/?theme=core).
2. Confirm you can navigate to '.../patterns/utility-actions/' on the doc site
3. Confirm the copy on the page matches the copy listed in [Confluence](https://confluence.cms.gov/pages/viewpage.action?pageId=1092192018) under the heading "Draft Guidance"

## Checklist


### This is a change to documentation:


### Screenshots:
With EmailIcon collision:
<img width="475" alt="Screenshot 2024-09-09 at 9 58 51 AM" src="https://github.com/user-attachments/assets/b40df53f-79cf-44f0-bad2-e98392b2a677">

Without EmailIcon collision:
<img width="769" alt="Screenshot 2024-09-09 at 12 39 41 PM" src="https://github.com/user-attachments/assets/2a11cbdd-7325-4f5d-a4eb-3dcda47ed279">
